### PR TITLE
Add stand-alone FSM node to handle WS requests

### PR DESCRIPTION
**Pull request by IvayloPapazov**
part of: https://github.com/aeternity/terraform-aws-testnet-api/issues/23

### CODE REVIEW

File: test/test.tf [hunk 1]
```diff
@@ -26,3 +26,34 @@ module "aws_deploy-test" {
   enable_state_channels = true
   enable_internal_api   = true
 }
+
+module "aws_deploy-test_vpc" {
+  source            = "../"
+  env               = "${var.env_name}"
```
Comment (venimus) [inline test/test.tf:32]: i think `${` stuff is deprecated

File: fleet/main.tf [hunk 2]
```diff
@@ -60,6 +60,12 @@ resource "aws_instance" "static_node" {
   }
 }
 
+resource "aws_lb_target_group_attachment" "test" {
```
[the remark this answers is not in the document]
Comment (IvayloPapazov) [reply fleet/main.tf]: 209bc53

File: main.tf [hunk 3]
```diff
@@ -10,8 +10,8 @@ module "aws_fleet" {
   envid  = "${var.envid}"
   kind   = "${var.kind}"
 
-  vpc_id  = "${module.aws_vpc.vpc_id}"
-  subnets = "${module.aws_vpc.subnets}"
+  vpc_id  = "${var.vpc_id != "" ? var.vpc_id : module.aws_vpc.vpc_id}"
```
[the remark this answers is not in the document]
Comment (IvayloPapazov) [reply main.tf:15]: 209bc53

File: fleet/templates/user_data.bash [hunk 4]
```diff
@@ -2,6 +2,8 @@
 # Don't enable debug mode in this script if secrets are managed as it ends-up in the logs
 set -eo pipefail
 
+sleep 10
```
[the remark this answers is not in the document]
Comment (IvayloPapazov) [reply fleet/templates/user_data.bash:6]: 209bc53

File: test/test.tf (hunk 1 above)
Comment (IvayloPapazov) [reply test/test.tf]: 209bc53

File: test/test.tf [hunk 5]
```diff
@@ -26,3 +26,35 @@ module "aws_deploy-test" {
   enable_state_channels = true
   enable_internal_api   = true
 }
+
+module "aws_deploy-test_vpc" {
+  source            = "../"
+  env               = var.env_name
+  envid             = "${var.envid}"
```
Comment (venimus) [inline test/test.tf:33]: Can you fix all "simple assignments" to not use "${}", as currently it is a mix of both. At least on the parts where you already did changes anyway

Comment (IvayloPapazov) [reply test/test.tf]: 82cada4

**Pull request merged**